internal/control_plane/server/api.go

```diff
@@ -1,31 +1,30 @@
 package server
 
 import (
+	"log"
 	"octavius/internal/config"
-	"octavius/internal/controlPlane/db/etcd"
-	"octavius/internal/logger"
-
-	// "crud-toy/internal/model"
+	"octavius/internal/control_plane/db/etcd"
+	"octavius/internal/control_plane/server/execution"
 	"octavius/pkg/protobuf"
 	"net"
-
 	"google.golang.org/grpc"
 )
 
 func Start() error {
 	appPort := config.Config().AppPort
-	listener, err := net.Listen("tcp", appPort)
+	listener, err := net.Listen("tcp", "localhost:"+appPort)
 	server := grpc.NewServer()
 	etcdClient := etcd.NewClient()
 	defer etcdClient.Close()
-	
+	exec := execution.NewExec(etcdClient)
+
 	procGrpcServer := NewProcServiceServer(exec)
-	procProto.RegisterProcServiceServer(server, procGrpcServer)
+	protobuf.RegisterProcServiceServer(server, procGrpcServer)
 	if err != nil {
-		logger.Fatal("grpc server not started")
+		log.Fatal("grpc server not started")
```

Comment:
if you do a fatal in here, the error is never returned at line 25. Please remove this log

Reply:
and also, please only return OR log the error.

Reply:
okay. cool.